classes/utils/Biome.py

```diff
@@ -0,0 +1,95 @@
+from enum import Enum
+
+
+class Biome(Enum):
+
+    def __init__(self, id, string):
+        self.id = id
+        self.string = string
+
+    def id_to_biome(id):
+        for biome in Biome:
+            if biome.id == id:
+                return biome
+        return None
+
+    def name_to_biome(name):
+        for biome in Biome:
+            if biome.name == name:
+                return biome
+        return None
+
+    def string_to_biome(string):
+        for biome in Biome:
+            if biome.biome == string:
+                return biome
+        return None
+
+    INVALID_BIOME = (-1, 'Invalid Biome')
```

Comment:
We should use 127 (VOID) instead so the application doesn't break. OR throw an error uppon encountering it